### ISSUE TITLE
Setting up arcpy_metadata on Windows 7

### ISSUE THREAD

**Issue by samisnunu** [ucd-cws/arcpy_metadata#14]
Hi,
I wonder if I can setup the arcpy_metadata on my windows 7 machine as where ArcGIS 10.2.2 and python 2.7 32-bit installed ?
Please advise,

Thanks,
Sami

**Comment by nickrsan** [comment]
Yes, that was its primary testing environment for the initial builds, in fact. Are you having trouble, or just checking in in advance?

**Comment by samisnunu** [comment]
Thanks Nick,

Yes I would like to test it and start working on it on my machine. but it seems I have to install other main python library/tools like setuptools, __version__, etc..
If there's a simple procedure for the setup workflow that I would be grateful.

**Comment by nickrsan** [comment]
Hi Sami,

If you've set up pip in your ArcGIS copy of Python, you should be able to do a simply "pip install arcpy_metadata" at the command line and have everything handled for you.

setuptools will be required by most packages, so it's a good idea to get that installed anyway. You can find installation info here: https://pypi.python.org/pypi/setuptools

**Comment by thomas-maschler** [comment]
Hey Sami,

we just published a new update which is not yet available using pip. The new update comes with many more metadata items and a much better API.
If you just want to test it, you can simply download the repo and copy the arcpy_metadata subfolder into your project. You should be able to call it using `import arcpy_metadata`. There are no other dependencies except arcpy.

**Comment by samisnunu** [comment]
Many Thanks Nick and Thomas!
in fact, I couldn't manage to install pip libraries on my machine, having issue with SSL security during the pip setup, not sure why or what is missing in their package..

But I followed your comments above, without installing pip, and managed to call the import arcpy_metadata. However I ran into python errors during doing simple test, so I'll close this question and raise another issue post in that regard.
